MetaMask/metamask-extension
# UX: Multichain: Add Account or Hardware Wallet button

## **Description**
Replaces the "Add / Import / Hardware" links with a single button that takes the user to another quick modal to add, import, or connect a hardware wallet.


## **Manual testing steps**

_1.  Open the account menu
_2. Click the "Add account or hardware wallet" button
_3. Click each option, ensure the correct modal displays
_4. Click "back" on each screen, ensure it goes back logically in the nested cycle

## **Screenshots/Recordings**


https://github.com/MetaMask/metamask-extension/assets/46655/7967d0ce-870c-4ccb-91bc-3f2e9cf55e79


## **Related issues**

_Fixes https://github.com/MetaMask/MetaMask-planning/issues/1374

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
